napari/layers/_tests/test_dask_layers.py

```diff
@@ -184,7 +186,7 @@ def test_dask_unoptimized_slicing(delayed_dask_stack, monkeypatch):
     v.dims.set_point(0, 3)
     # all told, we have ~2x as many calls as the optimized version above.
     # (should be exactly 8 calls, but for some reason, sometimes less on CI)
-    assert delayed_dask_stack['calls'] >= 7
+    assert delayed_dask_stack['calls'] >= 10
```

Comment:
Can you update the comment above about the expected number of calls?

Reply:
Done. also updated to set points relative to the initial point (which is no longer 0 since we start at the midpoint)